public/js/router.js

```diff
@@ -1,37 +1,28 @@
 'use strict';
 
-var Backbone = require( 'backbone' );
+var StateManager = require( 'stateman' );
 
-var analytics = require( './lib/analytics' );
-
-var Router = Backbone.Router.extend({
-
-  routes: {
-    '':               'index',
-    'green':          'green',
-    ':line':          'line',
-    ':line/:station': 'station',
-    '*notFound':      'error404'
-  },
-
-  index: require( './routes/index' ),
-
-  green: require( './routes/green' ),
-
-  line: require( './routes/line' ),
-
-  station: require( './routes/station' ),
+var router = new StateManager({
+  title: 'MBTAwesome'
+});
 
-  error404: require( './routes/404' ),
+router.state({
+  'home': require( './routes/index' ),
+  'green': require( './routes/green' ),
+  'line': require( './routes/line' ),
+  'station': require( './routes/station' ),
+  '404': require( './routes/404' )
+});
 
-  // Convenience wrapper for navigation (always sets { trigger: true })
-  goTo: function( target ) {
-    this.navigate( target, {
-      trigger: true
-    });
-    analytics.pageView();
-  }
+router.on( 'notfound', function handle404() {
+  router.go( '404', {
+    // Do not replace the URL, just load the 404 page within it
+    encode: false
+  });
+});
```

Comment:
There's a special [`$notfound` route](https://github.com/leeluolee/stateman/blob/master/src/stateman.js#L135). It's not documented tho', so it's a bit tough to learn about :)